pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -293,7 +293,7 @@ func TestLifeCycleHook(t *testing.T) {
 		testPod.DeletionGracePeriodSeconds = &gracePeriodLocal
 		testPod.Spec.TerminationGracePeriodSeconds = &gracePeriodLocal
 
-		m.killContainer(testPod, cID, "foo", "testKill", &gracePeriodLocal)
+		m.killContainer(testPod, cID, "foo", "testKill", time.Duration(gracePeriodLocal)*time.Second)
```

Comment:
Every time.Duration feels weird we never use the `time` itself , we cast it in int64 
This work but it's weird